src/manifest/extensions/tiles/TileExtensionType.ts

```diff
@@ -36,4 +36,9 @@ export enum TileExtensionType {
    * Admin Dashboard overview intelligent tile extension
    */
   ADMIN_DASHBOARD = 'tile-admin-dashboard',
+
+  /**
+   * Report strike zone tile extension
+   */
+  REPORT_STRIKE_ZONE = 'tile-report-strike-zone',
```

Comment:
this needs to be updated as well
https://github.com/getoutreach/extensibility-sdk/blob/main/docs/schema/2.0/manifest.schema.json#L510

Reply:
Hmm, @palootcenas-outreach if they are not going to create actual extensibility apps then the manifest schema does not need to be updated, right?

Reply:
Correct